modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonoidKLaws.kt

```diff
@@ -1,41 +1,44 @@
 package arrow.test.laws
 
 import arrow.Kind
+import arrow.core.extensions.eq
 import arrow.core.extensions.list.foldable.fold
+import arrow.test.generators.GenK
 import arrow.typeclasses.Applicative
 import arrow.typeclasses.Eq
+import arrow.typeclasses.EqK
 import arrow.typeclasses.MonoidK
 import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
 
 object MonoidKLaws {
 
+  fun <F> laws(SGK: MonoidK<F>, GENK: GenK<F>, EQK: EqK<F>): List<Law> =
+    laws(SGK, GENK.genK(Gen.int()), EQK.liftEq(Int.eq()))
+
   fun <F> laws(SGK: MonoidK<F>, AP: Applicative<F>, EQ: Eq<Kind<F, Int>>): List<Law> =
```

Comment:
Remove this completely. Using `Applicative.just` to create test data is awful.

Reply:
i have marked it as `@deprecated` and will remove it